Mathlib/RingTheory/PowerSeries/Inverse.lean

```diff
@@ -120,6 +120,14 @@ theorem sub_const_eq_X_mul_shift (φ : R⟦X⟧) :
 set_option linter.uppercaseLean3 false in
 #align power_series.sub_const_eq_X_mul_shift PowerSeries.sub_const_eq_X_mul_shift
 
+theorem invOfUnit_mul (φ : R⟦X⟧) (u : Rˣ) (h : constantCoeff R φ = u) :
```

Comment:
Same.

Reply:
Done. I also added a simp tag to be coherent with the mv case